contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -310,6 +342,8 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
             _saveOperatorSigningKeysStats(_nodeOperatorId, signingKeysStats);
```

Comment:
For `_initialize_v2()`:
- no event for `_setStuckPenaltyDelay()`
- event `ContractVersionSet` is redundant. Already emitted in `_setContractVersion()`.

Reply:
For `activateNodeOperator()`:
- shall we update stacked validators and totals in this method?

Reply:
> For `activateNodeOperator()`:
> 
> * shall we update stacked validators and totals in this method?

looks like no, as no other state updates are made except `active` so nothing to update in `totals`.
And opposite, in `deactivateNodeOperator` we should make an update